lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -781,6 +784,96 @@ def get(self, request, *args, **kwargs):
         return render_to_response(self.template_name, context)
 
 
+class MgmtCommands(SysadminDashboardView):
+    """This class provides a view to handle the execution of management commands"""
+
+    def get(self, request):
+        """Displays the view for the management commands tab"""
```

Comment:
I avoid single-line docstrings; if you ever need to update it
(particularly to add documentation for the function parameters), you'll
end up needing multiple lines anyway and this ensures a much more
legible diff.